cmd/ipfs/main.go

```diff
@@ -591,21 +590,37 @@ func profileIfEnabled() (func(), error) {
 	return func() {}, nil
 }
 
+var apiFileErrorFmt string = `Failed to parse %[1]s/api file.
+	error: %[2]s
+If there is no daemon running, it is safe to delete it.
+You can do it with:
+	ps aux | grep ipfs # check there is no ipfs daemon
```

Comment:
thanks @djdv, I think something like this works:

```
If you're sure go-ipfs isn't running, you can just delete it.
Otherwise check:
	ps aux | grep ipfs # on Linux and OSX
	tasklist | findstr ipfs # on Windows